tests/test_collection_loader.py

```diff
@@ -68,35 +68,48 @@
 """
 
 
+@pytest.fixture
+def tmp_collection_root():
```

Comment:
Could use the pytest builtin 'tmpdir' or 'tmp_path' fixtures here. (http://doc.pytest.org/en/latest/tmpdir.html)

Something vaguely like:

``` python

@pytest.fixture
def tmp_collection_root(tmpdir):
   collection_root = tmpdir.mkdir("ansible_collections").mkdir("placeholder_namespace").mkdir("placehold_name")
   return collection_root.strpath
```

But I wouldn't worry about it now. Maybe revisit the tmpdir handling and update to use the pytest fixtures later if useful.